xclim/sdba/measures.py

```diff
@@ -496,6 +500,13 @@ def _taylordiagram(
         }
     )
 
+    # Normalize the standard deviations byt the standard deviation of the reference.
+    if normalize:
+        out[{"taylor_param": [0, 1]}] = (
+            out[{"taylor_param": [0, 1]}] / out[{"taylor_param": 0}]
```

Comment:
If the field is homogeneous, yes.  I can't think of any case it would happen, and if so, doing a comparison with a Taylor diagram doesn't seem very interesting in this case? i.e. I would find this problematic even if `normalize=False` and we never do the division by 0.